tools/pythonpkg/src/native/python_conversion.cpp

```diff
@@ -489,32 +449,27 @@ PythonObjectType GetPythonObjectType(py::handle &ele) {
 	}
 }
 
-Value TransformPythonValue(py::handle ele, const LogicalType &target_type, bool nan_as_null) {
-	auto object_type = GetPythonObjectType(ele);
+struct PythonValueConversion {
+	static const LogicalType &ConversionTarget(Value &result, const LogicalType &target_type) {
+		return target_type;
+	}
 
-	switch (object_type) {
-	case PythonObjectType::None:
-		return Value();
-	case PythonObjectType::Bool:
-		return Value::BOOLEAN(ele.cast<bool>());
-	case PythonObjectType::Integer: {
-		Value integer;
-		if (!TryTransformPythonNumeric(integer, ele, target_type)) {
-			throw InvalidInputException("An error occurred attempting to convert a python integer");
-		}
-		return integer;
+	static void HandleNull(Value &result, const LogicalType &target_type) {
+		result = Value();
```

Comment:
Should this null not be typed?

Reply:
This was the behavior before - I just kept it